test/unit/requester-core.test.js

```diff
@@ -270,6 +290,42 @@ describe('requester util', function () {
             expect(requesterCore.getRequestOptions(request, options).lookup).to.be.a('function');
         });
 
+        // eslint-disable-next-line max-len
+        (IS_NODE ? it : it.skip)('should override lookup function and resolve localhost correctly when dnsOpts.all is true', function () {
+            var request = new sdk.Request({
+                    url: 'http://localhost:8080/random/path'
+                }),
+                options = requesterCore.getRequestOptions(request, {}),
+                hostname = 'localhost';
+
+            expect(options.lookup).to.be.a('function');
+
+            options.lookup(hostname, { all: true }, function (err, addresses) {
+                expect(err).to.be.null;
+                expect(addresses).to.be.an('array');
+                expect(addresses[0].address).to.be.oneOf(['127.0.0.1', '::1']);
+                expect(addresses[0].family).to.be.oneOf([4, 6]);
+            });
+        });
+
+        // eslint-disable-next-line max-len
+        (IS_NODE ? it : it.skip)('should override lookup function and resolve *.localhost correctly when dnsOpts.all is true', function () {
```

Comment:
what is expectation from code when is node is false ? why not assert that instead of skipping

Reply:
Browser cannot connect to sockets. There are few tests that are not applicable for browser. This is how tests are return in postman-runtime in general. Also the generic expectation of what happens in browser is already covered in other test cases.